### PR TITLE
Document about system dependencies (fix #531, fix #488, fix #474)

### LINKED ISSUE #474: Installing local dependencies errors on debian
## Detailed Description

When installing the project dependencies as describe in http://udata.readthedocs.io/en/latest/local-dependencies/#python-and-virtual-environment i got some errors that looks like 

1- Running setup.py bdist_wheel for lxml ... error
  Complete output from command /home/guest/Dev/udata/venv/bin/python2.7 -u -c "import setuptools, tokenize;**file**='/tmp/pip-build-xurgtv/lxml/setup.py';exec(compile(getattr(tokenize, 'open', open)(**file**).read().replace('\r\n', '\n'), **file**, 'exec'))" bdist_wheel -d /tmp/tmpfuUq8ipip-wheel- --python-tag cp27:
  Building lxml version 3.6.0.
  Building without Cython.
  ERROR: /bin/sh: 1: xslt-config: not found

  *\* make sure the development packages of libxml2 and libxslt are installed **
2-  creating build/temp.linux-x86_64-2.7/src/lxml
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -Isrc/lxml/includes -I/usr/include/python2.7 -c src/lxml/lxml.etree.c -o build/temp.linux-x86_64-2.7/src/lxml/lxml.etree.o -w
  In file included from src/lxml/lxml.etree.c:320:0:
  src/lxml/includes/etree_defs.h:14:31: fatal error: libxml/xmlversion.h: Aucun fichier ou dossier de ce type
   #include "libxml/xmlversion.h"
                                 ^
  compilation terminated.
  Compile failed: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  creating tmp
  cc -I/usr/include/libxml2 -c /tmp/xmlXPathInitztFWqH.c -o tmp/xmlXPathInitztFWqH.o
  /tmp/xmlXPathInitztFWqH.c:1:26: fatal error: libxml/xpath.h: Aucun fichier ou dossier de ce type
   #include "libxml/xpath.h"
                            ^
  compilation terminated.

---

  Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?

---

  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
3-Compile failed: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  creating tmp
  cc -I/usr/include/libxml2 -c /tmp/xmlXPathInitztFWqH.c -o tmp/xmlXPathInitztFWqH.o
  /tmp/xmlXPathInitztFWqH.c:1:26: fatal error: libxml/xpath.h: Aucun fichier ou dossier de ce type
   #include "libxml/xpath.h"
                            ^
  compilation terminated.

---

  Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?

---

  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
4-In file included from src/liblzma.c:1:0:
  src/liblzma.h:24:18: fatal error: lzma.h: Aucun fichier ou dossier de ce type
   #include <lzma.h>
                    ^
  compilation terminated.
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
## Context

When installing the project local dependencies

## Possible Implementation

I resolve it by doing
$sudo apt-get install build-essential pkg-config python-dev python-virtualenv libjpeg-dev zlib1g-dev libpng12-dev libtiff5-dev libfreetype6-dev liblcms2-dev libopenjpeg-dev libwebp-dev libpng12-dev libxml2-dev libxslt1-dev liblzma-dev

As suggested by @grischard on the gitter.
@davidbgk will add it to the documentation.

## Your Environment

- Debian Jessie (amd64):

### PR DESCRIPTION
**Pull request by noirbizarre**

### CODE REVIEW

File: docs/local-dependencies.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ $ pip install -r udata/requirements/develop.pip
 $ pip install -e udata/
 ```
 
+Some dependencies have an optionnal compilation support for Cython
+resulting in more performances (mostly XML harvesting).
```
Comment (davidbgk) [inline docs/local-dependencies.md]: Oh I didn't know about that!

File: docs/system-dependencies.md [hunk 2]
```diff
@@ -1,12 +1,57 @@
...
+## Git
 
-First of all, you need [Git][] and [Python][] 2.7 (installed by default on OSX and many Linux distributions).
+First of all, you need [Git][].
 
-If you use `apt-get`, the package is named `git-core`.
+If you use a Debian-like distribution with `apt-get`, the package is named `git-core`.
 If you prefer [Homebrew][] (OSX), the package is named `git`.
 
+## Python requirements
```
Comment (davidbgk) [inline docs/system-dependencies.md]: What about documenting why we do not use Python 3 in that section?

File: docs/local-dependencies.md (hunk 1 above)
Comment (noirbizarre) [inline docs/local-dependencies.md]: Saved me a lot of time before ;)

File: docs/system-dependencies.md (hunk 2 above)
Comment (noirbizarre) [inline docs/system-dependencies.md]: Good idea :+1:

Comment (noirbizarre) [inline docs/system-dependencies.md]: Done

Comment (davidbgk) [inline docs/system-dependencies.md]: 👍

**Pull request merged**